rust_src/src/marker.rs

```diff
@@ -1,12 +1,13 @@
 //! marker support
 
-use libc::ptrdiff_t;
+use libc::{c_void, ptrdiff_t};
 use std::mem;
+use std::ptr;
 
 use remacs_macros::lisp_fn;
-use remacs_sys::{BoolBF, EmacsInt, Lisp_Marker};
+use remacs_sys::{BoolBF, EmacsInt, Lisp_Buffer, Lisp_Marker};
 use remacs_sys::{buf_charpos_to_bytepos, mget_insertion_type, mset_insertion_type,
-                 set_marker_internal, set_point_both, Fmake_marker};
+                 set_marker_internal, set_point_both, unchain_marker, Fmake_marker};
```

Comment:
You meant this ?

Reply:
When I looked at this earlier, I had seen a `remacs_sys` import in with the local file imports. But I no longer see it.